### ISSUE TITLE
Rustc version 1.41 is not supported

### ISSUE THREAD

**Issue by Tiwalun** [probe-rs/target-gen#9]
When compiling with rustc version 1.41 under linux, the following error appears:

```
error[E0214]: parenthesized type parameters may only be used with a `Fn` trait
  --> /home/dominik/.cargo/registry/src/github.com-1ecc6299db9ec823/encoding_rs-0.6.11/src/utf_16.rs:33:44
   |
33 |     pub fn additional_from_state(&self) -> usize() {
   |                                            ^^^^^^^ only `Fn` traits may use parentheses
```

Active toolchain:
```
stable-x86_64-unknown-linux-gnu (default)
rustc 1.41.0 (5e1a79984 2020-01-27)
```

With rustc version 1.40 the crate can be compiled.

**Comment by Tiwalun** [comment]
This is the reverse dependency tree for the encoding_rs v0.6.11 crate:

```
encoding_rs v0.6.11
└── quick-xml v0.7.3
    ├── cmsis-pack v0.1.0 (https://github.com/probe-rs/cmsis-pack-manager.git#803b1e01)
    │   └── target-gen v0.1.0 (/home/dominik/Coding/target-gen)
    └── minidom v0.5.0
        └── cmsis-pack v0.1.0 (https://github.com/probe-rs/cmsis-pack-manager.git#803b1e01) (*)
```

**Comment by Yatekii** [comment]
Fixed in probe-rs/cmsis-pack-manager#2 :)
